README.md

```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
 }--->
 ```
 
+#### How to style examples
+
+You can’t reference external stylesheets when creating AMP pages. All styles must live in the head of the document using the tag `<style amp-custom>`.
```

Comment:
* s/creating AMP pages/creating samples/
* add: "AMP by Example provides a default styling for common elements (p, h1, h2, h3, a, ...) which you should use."